pkg/kindsys/kind.go

```diff
@@ -41,6 +41,10 @@ func (m Maturity) Less(om Maturity) bool {
 	return maturityIdx(m) < maturityIdx(om)
 }
 
+func (m Maturity) String() string {
```

Comment:
I thought exposing the `String() string` method here allows package's consumers be loosely coupled, instead of doing explicitly type (string) conversion here and there.

Reply:
ah yeah, that's fair